tests/Storage.test.ts

```diff
@@ -309,18 +309,19 @@ function putAllBlockData ()
     var idx = 0;
     var doPut = () =>
     {
+        if (idx >= sample_data.length)
+        {
+            getBlockData(1);
+                return;
```

Comment:
```suggestion
            getBlockData(1);
             return;
```

Also please add the explanation to the commit **message**, not just the PR description.

Reply:
Oh, I need to fix it, thank you.